From d92475a0a9fa0cf363189b2bd19bc232381a0a55 Mon Sep 17 00:00:00 2001
From: Bo Shen <voice.shen@atmel.com>
Date: Fri, 6 Mar 2015 16:59:21 +0800
Subject: [PATCH] Board: at91sam9rlek: remove 266MHz configuration

As the at91sam9rl SoC can not work more than 200MHz. So, remove
the configuration for 266MHz.

Signed-off-by: Bo Shen <voice.shen@atmel.com>
Acked-by: Nicolas Ferre <nicolas.ferre@atmel.com>
---
 board/at91sam9rlek/at91sam9rlek.h | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/board/at91sam9rlek/at91sam9rlek.h b/board/at91sam9rlek/at91sam9rlek.h
index eb8fe78..fcd4977 100644
--- a/board/at91sam9rlek/at91sam9rlek.h
+++ b/board/at91sam9rlek/at91sam9rlek.h
@@ -48,18 +48,6 @@
 
 #endif /* #ifdef CONFIG_CPU_CLK_200MHZ */
 
-#ifdef CONFIG_CPU_CLK_266MHZ
-
-#define MCK_133
-
-/* 133 MHz Bus clock */
-#define MASTER_CLOCK		(133000000)
-#define PLL_LOCK_TIMEOUT	1000000
-
-#define PLL_SETTINGS		0x2109BF0C
-
-#endif /* #ifdef CONFIG_CPU_CLK_266MHZ */
-
 /* Switch MCK on PLLA output PCK = PLLA = 2 * MCK */
 #define MCKR_SETTINGS		(AT91C_PMC_PRES_CLK | AT91C_PMC_MDIV_2)
 #define MCKR_CSS_SETTINGS	(AT91C_PMC_CSS_PLLA_CLK | MCKR_SETTINGS)
-- 
1.9.1

